root/etc/e-smith/templates/etc/httpd/admin-conf/httpd.conf/38ssl_settings

```diff
@@ -1,9 +1,4 @@
-#
-# 30ssl
-#
 
-# settings by https://cipherli.st/
-SSLCipherSuite EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH
 SSLProtocol All -SSLv2 -SSLv3
```

Comment:
Should this line be part of the legacy policy? In any case, I'd prefer to duplicate it if it's likely to be overridden by future policies

Reply:
`  SSLProtocol All -SSLv2 -SSLv3 `
added in all policy